main.css

```diff
@@ -21,6 +29,13 @@ html {
   margin: auto;
 }
 
+.backgroundDark {
```

Comment:
No hard rule, but some people consider it a good convention to use camel casing in JavaScript/TypeScript and use hyphens for naming in CSS. So `backgroundDark` can be renamed to `background-dark`.

Reply:
Got it thanks :)